app/data_model/questionnaire_store.py.orig

```diff
@@ -0,0 +1,61 @@
+import json
+import logging
+
+from app.data_model.answer_store import AnswerStore
+<<<<<<< HEAD
+from app.questionnaire.location import Location
+from app.storage.storage_factory import get_storage
+=======
+>>>>>>> Refactoring database storage to store user id/ik
+
+logger = logging.getLogger(__name__)
+
+
+class QuestionnaireStore:
+
+    def __init__(self, storage):
+        self.metadata = {}
+        self.answer_store = AnswerStore()
+        self.completed_blocks = []
+        self.storage = storage
+
+        if self.storage.has_questionnaire_state():
+            data = self.storage.get_user_data()
+
+            if 'METADATA' in data:
+                self.metadata = data['METADATA']
+
+            if 'ANSWERS' in data:
+                self.answer_store.answers = data['ANSWERS']
+
+            if 'COMPLETED_BLOCKS' in data:
+                for location_dict in data['COMPLETED_BLOCKS']:
+                    location = Location(
```

Comment:
Of course you could just leave Location as-is but allow init from a dict like above to make it easier to create one